doc/examples/example_hash.md

```diff
@@ -1,17 +1,16 @@
-Hash {#example3}
-====================
-Hash
-====
+# Hash
+
+# Hash
```

Comment:
Why does this document have two headlines at the same level with the same text?

Reply:
This is only due to the translation of the doxygen md flavour into a md flavour that is compatibale with github and doxygen. In a later commit example_hash.md is extended and the headlines are changed. See
https://github.com/bmwcarit/MoCOCrW/pull/78/commits/6caf6034be3790ef67fa39fb56b6e00591b99277#diff-eca6fc2d202f060d336bc10cced68961